### ISSUE TITLE
summary: Math inside the summary breaks it

### ISSUE THREAD

**Issue by sanklamm** [getpelican/pelican-plugins#784]
When I have a bit of math (wrapped with $-signs) around the summary block, the `<!-- PELICAN_BEGIN_SUMMARY -->` and `END_SUMMARY` get ignored.

An example:

```
## Problem

> The sum of the squares of the first ten natural numbers is,
> $\qquad 1^2 + 2^2 + ... + 10^2 = 385$
> The square of the sum of the first ten natural numbers is,
> $\qquad (1 + 2 + ... + 10)^2 = 55^2 = 3025$
> Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is $3025 − 385 = 2640$.
<!-- PELICAN_BEGIN_SUMMARY -->
> Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.
<!-- PELICAN_END_SUMMARY -->

## Next headline

Some text.
```

**Comment by jerryasher** [comment]
As food for thought, I am seeing the pelican-open_graph plugin break IF the content contains a reference to a relative path using {filename} notation. And I think that is also related to the summary memoization.

https://github.com/whiskyechobravo/pelican-open_graph/issues/10

The pelican-open_graph plugin operates when the signal content_object_init is sent.
It tries to obtain the summary.
But if the content contains a line like:

```
![a pelican appears]({filename}/images/pelican.jpg "a pelican appears") 
```

this message will be generated when the summary is requested:

```
WARNING: Unable to find `/images/pelican.jpg`, skipping url replacement.
```

**Comment by danmackinlay** [comment]
Is this fixed by #830?

**Comment by justinmayer** [comment]
Perhaps addressed via merged PR #973. If not, please comment here and I'll re-open.
